### ISSUE TITLE
Create the "Past Events" Page Layout

### ISSUE THREAD

**Issue by alfiyafatima09** [alfiyafatima09/v3_event_planners#3]
- Implement a basic layout for the "Past Events" page using Next.js and Tailwind CSS. 
- Include a section for each past event with placeholders for event details (e.g., event name, date, and location).

**Comment by Varshap2205** [comment]
Hey! would like to work on this issue
Can you please assign it to me

**Comment by alfiyafatima09** [comment]
Yes sure, but I need to know how are you going to organise the events ? Will teh Page be divided event based on occurrence wise. Do let me know the way you would feel optimal !
